keywords/MobileRestClient.py

```diff
@@ -377,7 +377,23 @@ def get_users(self, url, db, auth=None):
 
         return resp.json()
 
-    def create_user(self, url, db, name, password, channels=None, roles=None, auth=None):
+    def append_usr_collection_dict(self, user_scopes_collections, channels, scope, collection):
+        collection_dict = {"admin_channels": channels}
+        if scope in user_scopes_collections:
+            user_scopes_collections[scope][collection] = collection_dict
+        else:
+            user_scopes_collections[scope] = collection_dict
+        return user_scopes_collections
+
+    def delete_user(self, url, db, name, auth=None):
+        if auth:
+            resp = self._session.delete("{}/{}/_user/{}".format(url, db, name), auth=HTTPBasicAuth(auth[0], auth[1]))
+        else:
+            resp = self._session.delete("{}/{}/_user/{}".format(url, db, name))
+        log_r(resp)
+        resp.raise_for_status()
+
+    def create_user(self, url, db, name, password, channels=None, roles=[], auth=None, scope_dict=None):
```

Comment:
For comprehension can you rename the `scope_dict` argument to `collection_access`, as that is what it is defining